test/runtests.jl

```diff
@@ -71,3 +71,29 @@ end
         @test dir1_log == dir2_log
     end; end
 end
+
+# https://github.com/JuliaVersionControl/Git.jl/issues/51
+
+@testset "OpenSSH integration" begin
+    is_ci = parse(Bool, strip(get(ENV, "CI", "false")))
+    if is_ci
+        withtempdir() do sshprivkeydir
+            privkey_filepath = joinpath(sshprivkeydir, "my_private_key")
+            open(privkey_filepath, "w") do io
+                ssh_privkey = ENV["CI_READONLY_DEPLOYKEY_FOR_CI_TESTSUITE_PRIVATEKEY"]
+                println(io, ssh_privkey)
+            end # open
+            withenv("GIT_SSH_COMMAND" => "ssh -i \"$(privkey_filepath)\"") do
+                withtempdir() do workdir
+                    cd(workdir) do
```

Comment:
```suggestion
```

Reply:
Ah, I forgot that our own `withtempdir()` function does the `cd()` for us. I'll fix that.

Reply:
Fixed in 482f020fc22a086f6daeac83cfdb8a04f9092497

Reply:
Fixed in 7c53abcc3b9d300a4191251aa2df8180cc5d5432